src/service/video-manager-impl.js

```diff
@@ -493,6 +538,39 @@ class VideoEntry {
   }
 
   /**
+   * Called when the orientation of the device changes
+   * @param {boolean} isLandscape
+   * @private
+   */
+  orientationChanged_(isLandscape) {
+    // Put the video in/out of fullscreen depending on orientation
```

Comment:
Actually from what I've seen, when the video is fullscreen, `viewerForDoc(this.ampdoc_).isVisible()` always returns false. Is that the normal behavior or is it a bug?